rotkehlchen/chain/ethereum/modules/uniswap/v3/utils.py

```diff
@@ -211,11 +211,20 @@ def uniswap_v3_lp_token_balances(
         tokens_a, tokens_b = [], []
         for position in positions:
             try:
-                tokens_a.append(ethereum.get_basic_contract_info(to_checksum_address(position[2])))
-                tokens_b.append(ethereum.get_basic_contract_info(to_checksum_address(position[3])))
+                token_a_info = ethereum.get_basic_contract_info(to_checksum_address(position[2]))
+                token_b_info = ethereum.get_basic_contract_info(to_checksum_address(position[3]))
+                if token_a_info['decimals'] is None or token_b_info['decimals'] is None:
```

Comment:
Wait why? How can that happen? Didn't you just add code so that this never happens?

Reply:
My bad. I thought the query response cache was persistent.